content/blog/salesforce-vs-in-house-custom-development.md

```diff
@@ -0,0 +1,109 @@
+- We automated everything from day one, thanks to TDD and CI/CD - the build process was clearly defined and scripted, making it consistent and repeatable. This helped us improve our release frequency and lead time to release. It made our end users really happy as they saw the features from their wishlists in the product way sooner than expected
+- Testing and quality checks were automated, keeping manual testing to a minimum
+
+_Metamorphosis was always release-ready._
+
+## Graphic: Scoreboard
+
+| Salesforce Scoreboard | Metamorphosis Scoreboard |
+| --- | --- |
+| Users Onboarded - 100 | Users Onboarded - 800+ |
+| Operational CTC for 100 users = 270,000$ | Operational CTC per year for 800+ users = 100,000$ |
+| Typical Lead Time to Release = Undefined | Typical Lead Time to Release = 2 Weeks |
+
+## The Impact
+
+Genie started saving 4000 hours a day across 800 users.
+
+## What Incubyte Achieved
+
+- **Saved 4000 hours a day.** Team members onboarded by the Metamorphosis project have a seamless workflow that resulted in time saving
+- **Over 800 users onboarded** by maintaining smaller, stable, consistent releases
+- **It's a tailored solution**, it is more than customization, because Incubyte developed a solution with the end user and their use cases in mind.
+- **By being fast, economical, and agile**, within two years, Metamorphosis added value to the company 
+
+
+## Where Salesforce works
+
+In this case study, we have analysed the progress made by both these methods over the time period of two years for the company Genie. However, it would be wrong to assume that in-house software development and no-code tools are drop-in replacements for one another. They have their own strengths.
+
+A low-code CRM can be very powerful once built and launched well. They are ideal for running campaigns, marketing outreach, and more.
+
+CRMs like Salesforce would work well in a business model where the nature of things is very well defined but the moment multiple systems are involved with different data points, its power weakens. In a nuanced environment, with unique use cases and intricacies, it is just as hard to work with something like Salesforce as it is to develop a tailored software solution to your specifications.
+
+When it comes to adopting solutions like Salesforce, a company would need to pay for the portal (license) and pay separately for the additional development services the business needs. Whereas custom software shifts the cost from Op-ex to Cap-ex.
+
+Consider your requirements and choose wisely. Will your end users absolutely benefit from the CRM or could they do more with a custom tool?
+
+## The power of Metamorphosis, a.k.a. Custom Tailored Development
+- **Ecosystem built per your need** so keeps your business, client and employees in mind from day one.
+- **Cost shifts from opex to capex** once built you do not have to pay additional per user you add to the ecosystem
```

Comment:
Maybe change it to, once built there is no need for additional expenditure to onboard new users. You can choose to ignore this if you want to include per user keyword